web/frontend/src/components/slam.vue

```diff
@@ -228,170 +273,199 @@ const refresh3dMap = () => {
-                <div class="relative">
-                  <select
-                    v-model="selected3dValue"
-                    class="
-                      m-0 w-full appearance-none border border-solid border-black bg-white bg-clip-padding px-3 py-1.5
-                      text-xs font-normal text-gray-700 focus:outline-none
-                    "
-                    aria-label="Default select example"
-                    @change="selectSLAMPCDRefreshFrequency()"
-                  >
-                    <option value="manual">
-                      Manual Refresh
-                    </option>
-                    <option value="30">
-                      Every 30 seconds
-                    </option>
-                    <option value="10">
-                      Every 10 seconds
-                    </option>
-                    <option value="5">
-                      Every 5 seconds
-                    </option>
-                    <option value="1">
-                      Every second
-                    </option>
-                  </select>
-                  <div
-                    class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2"
-                  >
-                    <svg
-                      class="h-4 w-4 stroke-2 text-gray-700"
-                      viewBox="0 0 24 24"
-                      stroke="currentColor"
-                      stroke-linejoin="round"
-                      stroke-linecap="round"
-                      fill="none"
-                    >
-                      <path d="M18 16L12 22L6 16" />
-                    </svg>
-                  </div>
-                </div>
+                <p>{{ basePose.getX()}}</p>
+
+                <p class="pl-9 pr-2 text-xs text-gray-500">
+                  y
+                </p>
+                <p>{{ basePose.getY() }}</p>
+
+                <p class="pl-9 pr-2 text-xs text-gray-500">
+                  z
+                </p>
+                <p>{{ basePose.getZ() }}</p>
               </div>
-              <div class="px-2 pt-7">
-                <v-button
-                  v-if="show3d"
-                  icon="refresh"
-                  label="Refresh"
-                  @click="refresh3dMap()"
-                />
+            </div>
+            <div class="flex flex-col pl-10">
+              <p class="text-xs">
+                Current Orientation
+              </p>
+              <div class="flex flex-row items-center">
+                <p class="pr-2 text-xs text-gray-500">
+                  o<sub>x</sub>
+                </p>
+                <p>{{ basePose.getOX() }}</p>
+
+                <p class="pl-9 pr-2 text-xs text-gray-500">
+                  o<sub>y</sub>
+                </p>
+                <p>{{ basePose.getOY() }}</p>
+
+                <p class="pl-9 pr-2 text-xs text-gray-500">
+                  o<sub>z</sub>
+                </p>
+                <p>{{ basePose.getOZ() }}</p>
```

Comment:
[q] Does this still output a pose where the Y value is at 0 and the X/Z are the ones that change with position?

Reply:
I know we discussed this and that it should be something the slam team fixes on the cartographer end but wanted to confirm.

Reply:
> Does this still output a pose where the Y value is at 0 and the X/Z are the ones that change with position?

yep